src/cocotb/share/lib/fli/FliImpl.cpp

```diff
@@ -564,6 +564,11 @@ GpiObjHdl *FliImpl::get_root_handle(const char *name) {
     return NULL;
 }
 
+GpiObjHdl *FliImpl::get_package_handle(const char *name) {
+        LOG_ERROR("Failed to get package handle for %s . . . not implemented for FLI", name);
+        return NULL;
```

Comment:
I know that `vpiPackage` is a part of the SV VPI extension, so I'm guessing that the FLI will be ineligible.  Please correct me if I'm wrong.